android/build.gradle

```diff
@@ -7,6 +7,7 @@ buildscript {
 
     dependencies {
         classpath 'com.android.tools.build:gradle:3.5.0'
+        classpath 'com.google.gms:google-services:4.3.3'
```

Comment:
Our general google services dependency for firebase related items and google analytics if we choose to use it